site/de/_posts/2021-06-10-projects.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+lang: de
+lang-ref: Bewilligung OCR-D-Projekte
+title: "Implementierungs- und Modulprojekte bewilligt"
+tags: [DFG, grant, phase 3]
+author: Elisabeth Engl
+---
+
+Neben dem Antrag des Koordinierungsprojekts wurden von der DFG sieben Implementierungs- 
+und Modulprojekte bewilligt, die in den kommenden Monaten ihre Arbeit aufnehmen werden.
+
+
+Von den elf eingereichten Anträgen, die im Zuge einer umfangreichen Pilotierung der 
+OCR-D-Software im Sommer 2020 erarbeitet wurden, werden vier Implementierungs- und drei 
+Modulprojekte von der DFG gefördert. 
+
+Die Modulprojekte werden ausgewählte OCR-D-Werkzeuge weiterentwickeln.
+* Workflow für werkspezifisches Training auf Basis generischer Modelle mit OCR-D sowie Ground Truth Aufwertung (UB Mannheim)
+* Font Group Recognition for Improved OCR (JGU Mainz, FAU Erlangen)
+* OLA-HD Service - Ein generischer Dienst für die Langzeitarchivierung historischer Drucke (SUB Göttingen, GWDG)
+
+Zusammen mit den übrigen derzeit ca. 65 OCR-D-Werkzeugen bilden diese die Basis für die
+Arbeiten der vier Implementierungsprojekte, die die OCR-D-Software für
+den produktiven Einsatz in der Massendigitalisierung vorbereiten werden. 
+* Integration von Kitodo und OCR-D zur produktiven Massendigitalisierung (UB Braunschweig, SLUB Dresden, UB Mannheim)
+* OPERANDI: OCR-D Performance Optimisation and Integration (SUB Göttingen, GWDG)
+* OCR-D Software in modularen Massendigitalisierungs Workflows (ULB Halle)
+* OCR4all libraries Volltexterkennung historischer Sammlungen (GEI Braunschweig, HCI und ZPD der Universität Würzburg)
+
+Wir freuen uns sehr auf die (weitere) Zusammenarbeit mit den Implementierungs- und 
+Modulprojekten, die den Einsatz der OCR-D-Software in verschiedenen Nutzungsszenarien
+vorbereiten werden, und sind gespannt auf diese nächste Projektphase. 
```

Comment:
"Gespannt" klingt für mich nach "na mal gucken was uns da erwartet". Etwas optimisterisches würde ich vorschlagen. (Muss ich noch mal drüber nachdenken was.)

Reply:
Mir fällt kein besseres Wort ein, ohne "Freude" zu wiederholen. Ggf. den Teil weglassen. Oder du hast noch eine Idee.

Reply:
I had the same problem. wanted to write sth like "freuen" but simply couldn't think of a synonym... perhaps it's better to put the second part of the sentence first, then we go with just one verb